### PR TITLE
added button component

### LINKED ISSUE #3: Button Component
Untitled
- Props: text, link
- /src/components/Button.jsx
- Try make it inner shadow on hover
- Use Next Link

### PR DESCRIPTION
**Pull request by ssant096**
image

### CODE REVIEW

File: src/app/not-found.js
Comment (seanquiambao) [inline src/app/not-found.js:1]: restore the original `not-found.js` file to avoid merge conflicts

**Comment by seanquiambao** [comment]
don't forget to link your issue under development

File: src/app/not-found.js
Comment (ssant096) [reply src/app/not-found.js:1]: I restored the original not-found.js file

**Review by seanquiambao — APPROVED** [review]
great job!

**Pull request merged**